hat/assets/js/apps/Iaso/domains/dataSources/components/VersionsDialog.js

```diff
@@ -236,7 +241,10 @@ const VersionsDialog = ({ renderTrigger, source, forceRefreshParent }) => {
             renderActions={({ closeDialog }) => (
                 <DialogActions className={classes.action}>
                     <Button onClick={closeDialog} color="primary">
-                        <FormattedMessage id="iaso.label.close" />
+                        <FormattedMessage
+                            id="iaso.label.close"
+                            defaultMessage="Close"
```

Comment:
That smells of something a certain former backend dev would have done, or rather forgotten to do 😁